parsers/__init__.py

```diff
@@ -45,12 +56,17 @@ class ParserFormats(Enum):
     METAPSICOV = PsicovParser
     NEBCON = PsicovParser
     EVFOLD = EvfoldParser
+    CCMPRED = CCMpredParser
+    CASPRR = CasprrParser
 
 
 class ContactFormats(Enum):
     PSICOV = PsicovParser
     METAPSICOV = PsicovParser
     NEBCON = PsicovParser
+    EVFOLD = EvfoldParser
+    CCMPRED = CCMpredParser
+    CASPRR = CasprrParser
```

Comment:
I think CASPRR format is equivalent to PSICOV, so let's just do `CASPRR = PsicovParser`